aspnet/migration/rc2-to-rtm.rst

```diff
@@ -0,0 +1,56 @@
+Migrating from ASP.NET Core RC2 to ASP.NET Core 1.0
+===================================================
+
+By `Cesar Blum Silveira`_
+
+.. contents:: Sections:
+  :local:
+  :depth: 1
+
+Overview
+--------
+
+This migration guide covers migrating an ASP.NET Core RC2 application to ASP.NET Core 1.0.
+
+There weren't many significant changes to ASP.NET Core between the RC2 and 1.0 releases. For a complete list of changes, see the `ASP.NET Core 1.0 announcements <https://github.com/aspnet/announcements/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0.0>`_.
```

Comment:
fixed with
See the following resources for a list of some of the most significant changes, announcements and migrations information: